src/components/TextInput/styles.ts

```diff
@@ -0,0 +1,69 @@
+import { styled } from '@mui/material/styles';
+import { TextField } from '@mui/material';
+import { Icon } from '../Icon';
+
+const HelperTextErrorIcon = styled(Icon)`
+  color: ${({ theme }) => theme.palette.colors.error75};
+  max-height: 9.6px;
+  max-width: 9.6px;
+  margin: 0px 4px 4px 0px;
```

Comment:
pls dont use direct margin like this; prefer `theme.spacing`, let's chat when you see this